.clang-tidy

```diff
@@ -186,6 +189,7 @@ Checks: >-
   -readability-simplify-boolean-expr,
   -readability-suspicious-call-argument,
   -readability-uppercase-literal-suffix,
+  -ctuOneDefinitionRuleViolation
```

Comment:
I'm not sure that's a good idea.
Is it 100% false-positive? Otherwise it should be fixed, not suppressed.

Reply:
Yea these dont look like false positives.